lib/src/kbasesearchengine/search/ElasticIndexingStorage.java

```diff
@@ -1615,6 +1607,26 @@ private FoundHits queryHits(
         return ret;
     }
 
+    private List<Object> createSortingQuery(List<SortingRule> sorting) {
+
+        List<Object> sort = new ArrayList<>();
+        for (SortingRule sr : sorting) {
+            String keyProp;
+            if(sr.isWorkspaceId){
+                keyProp = R_OBJ_ACCESS_GROUP_ID;
+            }else if (readableNames.containsValue(sr.keyName)){
+                keyProp = sr.keyName;
+            }else{
+                keyProp = getKeyProperty(sr.keyName);
+            }
+            Map<String, Object> sortOrderWrapper = ImmutableMap.of(readableNames.inverse().get(keyProp),
```

Comment:
Surely if keyProp was produced by getKeyProperty it won't be in readableNames.inverse()?